tests/unit-core/test-typedarray.c

```diff
@@ -303,6 +303,44 @@ test_typedarray_complex_creation (test_entry_t test_entries[], /**< test cases *
   }
 } /* test_typedarray_complex_creation */
 
+/**
+ * Test get/set/delete property by index.
+ */
+static void test_property_by_index (test_entry_t test_entries[])
+{
+
+  jerry_value_t ja = jerry_create_number (3);
+  jerry_value_t jb = jerry_create_number (6);
+  jerry_value_t jc = jerry_create_number (50);
```

Comment:
We should improve this to test more values not just simple integers. We need negative numbers, zero, inf/-inf, double values. Having multiple values we can create an array of test values (similar to the test_entries array) thus we would not need multiple variables.